### ISSUE TITLE
Dark Mode

### ISSUE THREAD

**Issue by aboe026** [windirstat/windirstat#106]
It would be great if WinDirStat had a dark mode/theme to make it easier on the eyes when using it late at night.

**Comment by NoMoreFood** [comment]
I would be supportive of adding this feature but would really love someone to play around and see how far they can get.

**Comment by hueldoeu** [comment]
yes. dark mode is a must have. i have been banned on airbnb for invalid reasons but they still don't offer a dark mode therefore i must refuse on airbnb. dark mode is a useful setting.

whoever helps me with using their Govt ID to get a verified and non-banned airbnb account and keep all the money i earn. i just like to be a hospitable person.

**Comment by Neustradamus** [comment]
Very important feature!

**Comment by Denyerec** [comment]
I remember once upon a time long ago when you could change the colours in Windows, and all apps would honour them... Absolutely mad that developers have to create a customised scheme for something that should be governed by the OS :(

**Comment by Bleauhaus** [comment]
@NoMoreFood no skills to help here but I offer a strong vote for this

**Comment by clngr** [comment]
Great feature to have!

**Comment by b1k3rdude** [comment]
Just adding my 2 cents, I would also like the option of having Darkmod please :-)

**Comment by squeak9** [comment]
It's officially in the works now!! https://github.com/windirstat/windirstat/commit/53ff23d075393a812cf1ac8283592d774363daaa

Screenshots: https://github.com/windirstat/windirstat/issues/313

I guess you have to build it yourself right now but sounds like a new beta is coming soon: https://github.com/windirstat/windirstat/issues/313#issuecomment-3489002791

**Comment by NoMoreFood** [comment]
Can you please validate this build addresses your request / issue?

https://github.com/windirstat/windirstat/releases/tag/beta/v2.4.0/2025-11-07

**Comment by Neustradamus** [comment]
Good job, thanks to @harryytm, @NoMoreFood.

Linked to:
- https://github.com/windirstat/windirstat/issues/106
- https://github.com/windirstat/windirstat/issues/313
- https://github.com/windirstat/windirstat/pull/314
- https://github.com/windirstat/windirstat/pull/317
- https://github.com/search?q=repo%3Awindirstat%2Fwindirstat+Dark+Mode&type=commits

**Comment by Bleauhaus** [comment]
Excellent !!
My Dark Mode Campaign conquers yet another system! Good Job @Decision
Makers for putting in the effort to do this. I for one have worsening
Eyesight issues and this makes a big difference for me, And It Looks cool
while saving Electricity. Soon all Desktop Software,, Mobile Apps, and
 WebSites
will fall to the DarkSide

**Comment by hueldoeu** [comment]
> Good job, thanks to [@harryytm](https://github.com/harryytm), [@NoMoreFood](https://github.com/NoMoreFood).
> 
> Linked to:
> 
> * [Dark Mode #106](https://github.com/windirstat/windirstat/issues/106)
> * [Darkmode Feedback #313](https://github.com/windirstat/windirstat/issues/313)
> * [Fix checkbox label color in dark mode #314](https://github.com/windirstat/windirstat/pull/314)
> * [Fix Column Header Bar Drawing Glitch #317](https://github.com/windirstat/windirstat/pull/317)
> * https://github.com/search?q=repo%3Awindirstat%2Fwindirstat+Dark+Mode&type=commits

amazing work bro. congrats to everyone who is responsible for it

<img width="1414" height="764" alt="Image" src="https://github.com/user-attachments/assets/1a173e78-f96d-411a-94c2-8026990036fe" />

<img width="1418" height="765" alt="Image" src="https://github.com/user-attachments/assets/d87f0c0e-42cb-4445-ad17-ecd2dd740e83" />

**Comment by b1k3rdude** [comment]
Thank you for dark mode :-)

**Comment by Neustradamus** [comment]
It is possible to add the Dark mode for Windows before Windows 10?

WinRAR, Notepad++, for example, have added Dark mode.

**Comment by NoMoreFood** [comment]
@Neustradamus Possible, of course... but not really interested in going that far back given it's probably like less than %1 of the folks who use WinDirStat and it's not an app that people run as frequently as WinRAR and Notepad++.

**Comment by NoMoreFood** [comment]
Closing due to release of https://github.com/windirstat/windirstat/releases/tag/release/v2.5.0.  If you have any issues with this new release, please open a new issue. Thanks!

**Comment by hueldoeu** [comment]
> Closing due to release of https://github.com/windirstat/windirstat/releases/tag/release/v2.5.0. If you have any issues with this new release, please open a new issue. Thanks!

this software is a masterpiece. will v2.5.0 be a non beta but release candidate a.k.a. stable?

**Comment by NoMoreFood** [comment]
@hueldoeu Yes, it's officially non-beta.
